suchmememanyskill/TegraExplorer
# automation (wow)

- tsv3 start ig
- optimize arrayClass.c
- implement if.else()
- make the garbage collector less terrible
- add break
- add dicts
- add string array indexing
- implement the most important function
- frii memory leak fix
- add scripts to the main menu, fix script parser reading beyond EOF
- remove old gc code
- properly implement #REQUIRE (VER x.x.x, MINERVA, KEYS)
- add embedding TE scripts
- safety push
- add crude firmware dumping script
- Fix line counter maybe?
- possibly fix parser instability??
- safety push
- build properly after clean on windows
- add comments, fix std bugs, add menu
- make fwdump.te significantly more cursed
- F to Dump Firmware
- fuk
- make Systemwipe.te
- remove pauses
- a
- [Untested] implement some more fs functions
- add str.split
- makefile fix for linux?
- add save.commit() and save.write()
- add #define SCRIPT_ONLY
- comments should count as lines
- automation is here
